src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Media_Animation/DoubleAnimation_Tests.TransformGroup.cs

```diff
@@ -0,0 +1,496 @@
+						{ color, defaultColor},
+						{ defaultColor, default},
+					})
+					.WithTolerance(192), // known to be up to 128 ...
+
+				ExpectedPixels
+					.At("Bottom left", host.X - 1, host.Bottom - 3)
+					.Pixels(new[,]
+					{
+						{ defaultColor, color},
+						{ defaultColor, default},
+						{ defaultColor, default},
+						{ defaultColor, default},
+						{ defaultColor, defaultColor}
+					})
+			);
+		}
+
+		[Test]
+		[AutoRetry]
+		public void When_TransformGroup_Composite()
+		{
+			const string color = color5;
+			const int trY = 50;
+
+			var (host, scale, half, final) = BeginTransformGroupTest("CompositeHost");
+
+			// "Half" is approximative, we only validate that the element has move bellow
+			ImageAssert.HasPixels(
+				half,
+
+				ExpectedPixels
+					.At("Top left", host.X - 1, host.Y - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, defaultColor},
+						{ defaultColor, bgColor},
+					}),
+
+				ExpectedPixels
+					.At("Top right", host.Right - 1, host.Y - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, defaultColor},
+						{ bgColor, defaultColor},
+					}),
+
+				ExpectedPixels
+					.At("Bottom right", host.Right - 1, host.Bottom - 1)
+					.Pixels(new[,]
+					{
+						{ color, defaultColor},
+						{ color, defaultColor},
+					}),
+
+				ExpectedPixels
+					.At("Bottom left", host.X - 1, host.Bottom - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, color},
+						{ defaultColor, color},
+					})
+			);
+
+			// Assert the final state
+			var finalTrY = trY * scale;
+			ImageAssert.HasPixels(
+				final,
+
+				ExpectedPixels
+					.At("Top left", host.X - 1, host.Y + finalTrY - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, bgColor},
+						{ defaultColor, color}
+					}),
+
+				ExpectedPixels
+					.At("Top right", host.Right - 1, host.Y + finalTrY - 1)
+					.Pixels(new[,]
+					{
+						{ bgColor, defaultColor},
+						{ color, defaultColor},
+					}),
+
+				ExpectedPixels
+					.At("Bottom right", host.Right - 1, host.Bottom + finalTrY - 1)
+					.Pixels(new[,]
+					{
+						{ color, defaultColor},
+						{ defaultColor, default},
+					}),
+
+				ExpectedPixels
+					.At("Bottom left", host.X - 1, host.Bottom + finalTrY - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, color},
+						{ default, defaultColor},
+					})
+			);
+		}
+
+		[Test]
+		[AutoRetry]
+		public void When_TransformGroup_Nested()
+		{
+			const string color = color6;
+
+			var (host, scale, half, final) = BeginTransformGroupTest("CrazyHost");
+
+			// "Half" is approximative, we only validate that the element is no longer in top left and is flowing bottom right
+			ImageAssert.HasPixels(
+				half,
+
+				ExpectedPixels
+					.At("Bottom left", host.X - 1, host.Y - 1)
+					.Pixels(new[,]
+					{
+						{ defaultColor, defaultColor},
+						{ defaultColor, bgColor},
+					})
+
+				//ExpectedPixels
+				//	.At("Bottom right", host.Right - 1, host.Bottom - 1)
+				//	.Pixels(new[,]
+				//	{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4919453)